src/API/Plugins.php

```diff
@@ -431,6 +445,48 @@ public function activate_plugins( $request ) {
 		) );
 	}
 
+	/**
+	 * Return recommended payment plugins.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return \WP_Error|\WP_HTTP_Response|\WP_REST_Response
+	 */
+	public function recommended_payment_plugin( $request ) {
+		// Default to marketing category (if no category set).
+		$all_plugins        = PaymentPlugins::get_instance()->get_recommended_plugins();
+		$valid_plugins      = [];
+		$per_page           = $request->get_param( 'per_page' );
+		$locale             = get_locale();
```

Comment:
I'm not 100% sure locale is the correct value to use for comparison here (IIRC that can differ from the store's location). AFAIK we've been using `wc_get_base_location()` elsewhere for checking what country a store is in.

Reply:
This is referring to the language though, as the json data is in english, which is why we only support english locales.

Reply:
Ah gotcha, I didn't catch that.